frontend/ui/widget/numberpickerwidget.lua

```diff
@@ -146,7 +146,51 @@ function NumberPickerWidget:init()
                             text = _("OK"),
                             is_enter_default = true,
                             callback = function()
-                                local input_value = tonumber(input_dialog:getInputText())
+                                local input_text = input_dialog:getInputText()
+                                local input_value = tonumber(input_text)
+                                local turn_off_checks = false
+
+                                -- if the first character of the input string is ":" turn of min-max checks
```

Comment:
`,` turn `off`
(I think `!` might be more explicite than `:` for this, meaning "attention!", but it might be confused with `1`)